src/dmd/dmodule.d

```diff
@@ -545,7 +545,7 @@ extern (C++) final class Module : Package
     Identifiers* versionidsNot; // forward referenced version identifiers
 
     MacroTable macrotable;      // document comment macros
-    Escape* escapetable;        // document comment escapes
+    Escape* _escapetable;       // document comment escapes
```

Comment:
although the chance that someone one day touches this directly are low it should be set `private ` if possible.